src/ImageSharp/ImageFrame{TPixel}.cs

```diff
@@ -216,19 +216,10 @@ internal ImageFrame(Configuration configuration, ImageFrame<TPixel> source)
         /// It is imperative that the accessor is correctly disposed off after use.
         /// </remarks>
         /// </summary>
-        /// <returns>The <see cref="PixelAccessor{TPixel}"/></returns>
-        internal PixelAccessor<TPixel> Lock()
+        /// <returns>The <see cref="Buffer2D{TPixel}"/></returns>
+        internal Buffer2D<TPixel> Lock()
```

Comment:
`Lock` is an odd name now that we are not pinning. Wouldn't `GetRootFramePixelBuffer()` also call this? Is it required anymore?

Reply:
Forgot to remove this, good catch!